scripts/slicing-node-logs/slice.sh

```diff
@@ -0,0 +1,80 @@
+# As of 2022 June 13, these four events have risingEdge variants.
+#  ChainSyncServerEvent.TraceChainSyncServerUpdate
+#  TraceAddBlockEvent.AddedBlockToQueue
+#  TraceAddBlockEvent.PipeliningEvent.SetTentativeHeader
```

Comment:
FTR: `PipeliningEvent` is being elided since https://github.com/input-output-hk/cardano-node/pull/4038, but it does not affect these scripts directly, as they currently don't make use of this event.